app/services/events.js

```diff
@@ -44,8 +44,27 @@ GameUpApp.factory('events', ['$q', function($q){
     // deferred.resolve;
     // return deferred.promise;
     return gamesList;
+  }
 
+  service.getEvent = function(eventId){
+    var eventObj = Parse.Object.extend("Event");
```

Comment:
This is the default parse way of query the DB but we are using the parse angular library.

We should delete line 50 & 51 and replace it with:
var query = new Parse.Query("Event");